mos.yml

```diff
@@ -44,6 +45,7 @@ config_schema:
   - ["wifi.sta.gw", "s", {title: "Static Default Gateway"}]
   - ["wifi.sta.nameserver", "s", {title: "DNS Server"}]
   - ["wifi.sta.dhcp_hostname", "s", {title: "Host name to include in DHCP requests"}]
+  - ["wifi.sta.all_chan_scan", "b", false, {title: "true to scan all channels or false to perform fast scan"}]
```

Comment:
please call this `wifi.sta_all_chan_scan` the difference being that it doesn't get triplicated in the sta1 and sta2 sections and it really applies to all scans and is not specific to station profile

Reply:
@rojer the problem I see with that is that wifi cfg isn't available in the `mgos_wifi_dev_setup` function?

Reply:
`mgos_sys_config_get_wifi_sta_all_chan_scan()`